core/math/projection.cpp

```diff
@@ -93,13 +93,23 @@ Vector4 Projection::xform_inv(const Vector4 &p_vec4) const {
 			columns[3][0] * p_vec4.x + columns[3][1] * p_vec4.y + columns[3][2] * p_vec4.z + columns[3][3] * p_vec4.w);
 }
 
-void Projection::adjust_perspective_znear(real_t p_new_znear) {
-	real_t zfar = get_z_far();
-	real_t znear = p_new_znear;
+void Projection::adjust_perspective_znear_zfar(real_t p_new_znear, real_t p_new_zfar) {
+	real_t deltaZ = p_new_zfar - p_new_znear;
```

Comment:
```suggestion
	real_t delta_z = p_new_zfar - p_new_znear;
```
I see that this variable was called in camelCase before, but... why?